core/src/test/java/ai/timefold/solver/core/impl/score/stream/collector/bi/InnerBiConstraintCollectorsTest.java

```diff
@@ -1095,6 +1096,34 @@ public void consecutiveUsage() {
         assertResult(collector, container, buildConsecutiveUsage());
     }
 
+    @Override
+    @Test
+    public void loadBalance() {
+        var collector = ConstraintCollectors.<String, Integer, String> loadBalance((a, b) -> a, (a, b) -> b);
+        var container = collector.supplier().get();
+
+        // Default state.
+        assertUnfairness(collector, container, BigDecimal.ZERO);
+        // Add first value, we have one now.
+        var firstRetractor = accumulate(collector, container, "A", 2);
+        assertUnfairness(collector, container, BigDecimal.ZERO);
+        // Add second value, we have two now.
+        var secondRetractor = accumulate(collector, container, "B", 1);
+        assertUnfairness(collector, container, BigDecimal.valueOf(0.707107));
```

Comment:
Maybe explain in a comment this is `sqrt((3-2.5)^2 + (2 - 2.5)^2) = sqrt(0.5^2 + 0.5^2) = sqrt(0.25 + 0.25) = sqrt(0.5)`